### PR TITLE
update text on dd214 upload page

### PR DESCRIPTION
**Pull request by rianfowler**
screen shot 2018-01-26 at 10 40 23
Couldn't figure out an option to change the text on the Upload button to match the design.

### CODE REVIEW

File: src/js/vic-v2/components/DD214Description.jsx [hunk 1]
```diff
@@ -3,8 +3,12 @@ import React from 'react';
...
   return (
     <div>
-      <p>Please upload a copy of your DD214 to verify that you were honorably discharged.</p>
-      File types you can upload: PDF, JPG, PNG
+      <p>Please upload a copy of your military discharge papers, for example, a DD214, DD256, DD257, or NGB22, to verify that you were honorably discharged.</p>
+      <p>If you have more than one discharge document, please upload the one with the highest character of discharge.</p>
+      <p>If you don't have a digital copy of your discharge document, you can scan or take a photo of it and then upload it to your computer or phone. </p>
+
+      <div>File types you can upload: .jpg, .pdf, or .png</div>
+      <div>Maximum file size: (TBD)</div>
```
Comment (rianfowler) [inline src/js/vic-v2/components/DD214Description.jsx]: Says TBD in the design.

Comment (cehsu) [reply src/js/vic-v2/components/DD214Description.jsx]: I remember us talking about avoiding maximum file sizes, was that in reference to this field @jbalboni @stacyhlinn @lalexanderson-dsva ?

File: src/js/vic-v2/components/DD214Description.jsx [hunk 2]
```diff
@@ -3,8 +3,12 @@ import React from 'react';
 export default function DD214Description() {
   return (
     <div>
-      <p>Please upload a copy of your DD214 to verify that you were honorably discharged.</p>
-      File types you can upload: PDF, JPG, PNG
+      <p>Please upload a copy of your military discharge papers, for example, a DD214, DD256, DD257, or NGB22, to verify that you were honorably discharged.</p>
+      <p>If you have more than one discharge document, please upload the one with the highest character of discharge.</p>
+      <p>If you don't have a digital copy of your discharge document, you can scan or take a photo of it and then upload it to your computer or phone. </p>
```
Comment (cehsu) [inline src/js/vic-v2/components/DD214Description.jsx]: We tend to use curly apostrophes (’) in general.

**Review by cehsu — APPROVED** [review]
LGTM. Pending the feedback about the maximum file size question.

**Pull request merged**